src/shared/schedule_scan.c

```diff
@@ -108,9 +108,14 @@ int sched_scan_read(sched_scan_config *scan_config, xml_node **nodes, const char
     return _sched_scan_validate_parameters(scan_config);
 }
 
-time_t sched_scan_get_time_until_next_scan(sched_scan_config *config, const char *MODULE_TAG,  const int run_on_start) {
-    const time_t next_time = _get_next_time(config, MODULE_TAG, run_on_start); 
-    config->next_scheduled_scan_time = time(NULL) + next_time;
+time_t sched_scan_get_time_until_next_scan(sched_scan_config *config, const char *MODULE_TAG,  const int run_on_start, 
```

Comment:
@danimegar Please let's insert the daylight status parameter into the structure `sched_scan_config`.